src/mono/wasm/runtime/web-socket.ts

```diff
@@ -58,17 +58,20 @@ export function ws_wasm_create(uri: string, sub_protocols: string[] | null, rece
     ws.binaryType = "arraybuffer";
     const local_on_open = () => {
         if (ws[wasm_ws_is_aborted]) return;
+        if (loaderHelpers.is_exited()) return;
```

Comment:
Should this be before `if (ws[wasm_ws_is_aborted]) return` too?

Reply:
doesn't really matter.